src/pages/landing/[landingPage]/[article]/index.tsx

```diff
@@ -18,6 +18,7 @@ const LandingPageArticle = ({
   return (
     <>
       <PageHeader searchDisplay={false}>
+        <h1>{breadcrumbNavItems[2].toUpperCase()}</h1>
```

Comment:
I wasn't sure about adding this h1 here, but I thought displaying what landing page the article is nested under could be beneficial.